content/events/2021-09-27-workshop-caladaptr.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Working with Cal-Adapt Data in R"
+date: 2021-10-01
+location: Online
+time: "10:00 AM - 11:30 AM PST"
+tags: workshop 
+image: webinar.jpg (http://igis.ucanr.edu/Training/CaladaptROct21/)
```

Comment:
@mukhtyar did you mean to add a URL here?

Reply:
Good catch! Thought I'd removed that url.